scripts/atlas/README.md

```diff
@@ -3,79 +3,79 @@
 The [ESM Metagenomic Atlas](https://esmatlas.com) is a repository of over database of more than 600 million metagenomic protein structures predicted by ESMFold.
```

Comment:
do we want to update the `more than 600 million` stat? there are over 760M structures now

```
>>> len(df[~df.plddt.isna())
762985218
```
^ the actual count would be a bit higher as the backfill is still ongoing